docker_files/Dockerfile_postgres

```diff
@@ -0,0 +1,2 @@
+FROM postgres:15-alpine
+COPY db.sql /docker-entrypoint-initdb.d/
```

Comment:
This line is to instruct the image to use the sql file to pre-seed the database.